### PR TITLE
Fix logs path

### PR DESCRIPTION
**Pull request by ccqpein**
### Background

In [README](https://github.com/Torantulino/Auto-GPT#logs), it says the logs folder is located on `./logs`. However, it is `../logs`

### Changes

Change the path of logs folder.

### Documentation

None

### Test Plan

None

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Review by grg124 — APPROVED** [review]
LGTM

File: scripts/logger.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 class Logger(metaclass=Singleton):
     def __init__(self):
         # create log directory if it doesn't exist
-        log_dir = os.path.join('..', 'logs')
+        log_dir = os.path.join('.', 'logs')
```
Comment (p-i-) [inline scripts/logger.py:27]: Isn't this gona create scripts/logs/ ?
Where's `'.'`? Is that GUARANTEED to be scripts/ seeing as logger.py is in scripts/

Comment (p-i-) [reply scripts/logger.py:27]: I see this relative-file-location issue cropping up in a few places. Maybe we need something like 
```
script_folder = os.path.dirname(os.path.abspath(__file__))
proj_root_folder = os.path.dirname(script_folder)
```

... but if we need this in multiple places, better put some `getProjectRoot` func in utils.py or something

**Comment by richbeales** [comment]
closing in favour of https://github.com/Torantulino/Auto-GPT/pull/1312

**Pull request closed without merging**